tests/lcs/core/engine/put.livecodescript

```diff
@@ -16,6 +16,38 @@ for more details.
 You should have received a copy of the GNU General Public License
 along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
 
+
+on _TestPutBeforeInvalidContainer
+    put "foo" before card 1 of this stack
+end _TestPutBeforeInvalidContainer
+
+on _TestPutIntoInvalidContainer
+    put "foo" into card 1 of this stack
+end _TestPutIntoInvalidContainer
+
+on _TestPutAfterInvalidContainer
+    put "foo" after card 1 of this stack
+end _TestPutAfterInvalidContainer
+
+on TestPutBeforeIntoAfterInvalidContainer
+	TestAssertThrow "put before invalid container" , "_TestPutBeforeInvalidContainer", the long id of me, \
```

Comment:
Currently this is only testing that a card is an invalid container - we also need to test stack, widget, scrollbar, group, player, graphic and eps.

Reply:
ok sure :)